tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartPersonName.java

```diff
@@ -18,7 +18,6 @@
 
 public class ChartPersonName extends Chart {
     private static final CLDRConfig CLDR_CONFIG = CLDRConfig.getInstance();
-    static final Factory FACTORY = CLDR_CONFIG.getCldrFactory();
     static final CLDRFile ENGLISH = CLDR_CONFIG.getEnglish();
```

Comment:
Hmmm. if the FACTORY isn't right when coming for CLDR_CONFIG, then is the getEnglish()?

Reply:
yes, because en.xml is static. getEnglish within ST also loads the disk file.